packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -64,9 +64,27 @@ The third generation of the Okta Sign-In Widget doesn’t guarantee the stabilit
 
 * If you use an Okta-managed TLS certificate, you don't need a [Certificate Authority Authorization (CAA)](https://datatracker.ietf.org/doc/html/rfc6844) record. However, if you do have a CAA record, keep the following in mind:
 
-    * If it's your first time setting up a custom domain with an Okta-managed certificate, you need to add `letsencrypt.org` to the issuers list or Okta can't get the TLS certificate. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).
+  * If it's your first time setting up a custom domain with an Okta-managed certificate, you need to add `letsencrypt.org` to the issuers list or Okta can't get the TLS certificate. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).
 
-    * If you have an Okta-managed certificate and you later get a CAA record, Okta can't renew your certificate. You must either add `letsencrypt.org` to the issuers list or remove the CAA record.
+  * If you have an Okta-managed certificate and you later get a CAA record, Okta can't renew your certificate. You must either add `letsencrypt.org` to the issuers list or remove the CAA record.
+
+* If you use your own TLS certificate, keep in mind the following:
+
+  * It should be signed with the SHA-256 hash algorithm.
+
+  * It shouldn't be expired.
+
+  * Its start date shouldn't be in the future.
+
+  * Its expiration date can't be further than 10 years from now.
+
+  * It mustn't be self-signed.
+
+  * It must be a certificate for the actual domain.
+
+  * The public key must use the RSA algorithm.
```

Comment:
is there a particular RSA algorithm that must be used?

Reply:
The statement is correct - the certificate's public key must use RSA.

Here are all the validation messages we can return when validating the certificate: https://github.com/atko-eng/okta-core/blob/master/resources/src/main/resources/messages-untranslated.properties#L6331-L6354

Reply:
Thank you, Gorman :)